### PR TITLE
Friendly iframes: a more precise document.ready signal

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/friendly-iframe-embed.js [hunk 1]
```diff
@@ -96,36 +98,75 @@ export function installFriendlyIframeEmbed(iframe, container, spec) {
...
 }
 
 
 /**
+ * @param {!HTMLIFrameElement} iframe
+ * @return {boolean}
+ */
+function isIframeReady(iframe) {
+  const childDoc = iframe.contentWindow && iframe.contentWindow.document;
+  return (childDoc && isDocumentReady(childDoc) &&
```
Comment (cramforce) [inline src/friendly-iframe-embed.js]: Add docs explaining why this is complicated

File: src/friendly-iframe-embed.js [hunk 2]
```diff
@@ -96,36 +98,75 @@ export function installFriendlyIframeEmbed(iframe, container, spec) {
...
+        if (isIframeReady(iframe)) {
+          resolve();
+          win.clearInterval(interval);
+        }
+      }, /* milliseconds */ 5);
+
+      // For safety, make sure we definitely stop polling when child doc is
+      // loaded.
+      loadedPromise.catch(error => {
+        rethrowAsync(error);
```
Comment (cramforce) [inline src/friendly-iframe-embed.js]: clear here?

Comment (dvoytenko) [inline src/friendly-iframe-embed.js]: I'm clearing everything in the `then` below.

File: src/friendly-iframe-embed.js (hunk 1 above)
Comment (dvoytenko) [inline src/friendly-iframe-embed.js]: Done

**Pull request merged**